cusub_cortex/state_machine/src/tasks/task.py

```diff
@@ -426,14 +444,31 @@ def set_new_time(self, seconds):
         self.timed_out = False
         if seconds != 0:
             self.timer = rospy.Timer(rospy.Duration(seconds), self.timer_callback)
+            self.smprint("Next task time: " + str(seconds) + "s")    
         else:
-            rospy.logwarn("No timeout monitoring on next task")
+            self.smprint("No timeout monitoring on next task", warn=True)
 
     def timer_callback(self, msg):
         self.timer.shutdown()
-        rospy.logerr("Task Timed Out")
+        self.smprint("Task Timed Out", warn=True)
         self.timed_out = True
 
     def timed_out(self):
         """ In objectives reference like: userdata.timeout_obj.timed_out """
-        return self.timed_out
\ No newline at end of file
+        return self.timed_out
+
+    def smprint(self, string, warn=False):
+        if warn:
+            rospy.loginfo("[" + bcolors.OKGREEN + self.name + bcolors.ENDC + "] " + bcolors.WARNING +"[WARN] "+ string + bcolors.ENDC)
+        else:
+            rospy.loginfo("[" + bcolors.OKGREEN + self.name + bcolors.ENDC + "] " + string)
+
+class bcolors: # For terminal colors
+    HEADER = '\033[95m'
+    OKBLUE = '\033[94m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
```

Comment:
This is defined 4 different times, wouldn't it make sense to put it into its own file and import it everywhere that uses it, so that we would only have to change it in one place?

Reply:
Yeah completely agree